src/Core/Managed/Shared/Channel/Transmission.cs

```diff
@@ -57,12 +57,25 @@ public Transmission(Uri address, byte[] content, string contentType, string cont
             this.ContentEncoding = contentEncoding;
             this.Timeout = timeout == default(TimeSpan) ? DefaultTimeout : timeout;
             this.Id = Convert.ToBase64String(BitConverter.GetBytes(WeakConcurrentRandom.Instance.Next()));
+            this.TelemetryItems = null;
 #if CORE_PCL
             this.client = new HttpClient() { Timeout = this.Timeout };
 #endif
         }
 
-        internal Transmission(Uri address, IEnumerable<ITelemetry> telemetryItems, string contentType, string contentEncoding, TimeSpan timeout = default(TimeSpan)) 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Transmission"/> class.
+        /// </summary>
+        public Transmission(Uri address, ICollection<ITelemetry> telemetryItems, TimeSpan timeout = default(TimeSpan)) 
+            : this(address, JsonSerializer.Serialize(telemetryItems, true), "application/x-json-stream", JsonSerializer.CompressionType, timeout)
```

Comment:
This was effectively always the case, the serializing just now happens in this constructor directly. What would it even do if you gave a different content type? We only serialize to JSON.